ITunesSearch/ITunesSearch/SearchResultsTableViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SearchResultsTableViewController.swift
+//  ITunesSearch
+//
+//  Created by Taylor Lyles on 5/14/19.
+//  Copyright © 2019 Taylor Lyles. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultsTableViewController: UITableViewController, UISearchBarDelegate {
+	
+	let searchResultsController = SearchResultController()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+		
+		searchBar.delegate = self
+	}
+	
+	func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+		
+		guard let searchTerm = searchBar.text else { return }
```

Comment:
Would be a good idea to guard for if the user pass in an empty string. 

guard let searchTerm = searchBar.text, searchTerm != ""  else { return }